### ISSUE TITLE
Use of new Geocoder api

### ISSUE THREAD

**Issue by rduivenvoorde** [Lytrix/pdokbaggeocoder#3]
Hi, 

since 1 april the api used in the plugins is not used anymore.

See: 
https://www.pdok.nl/nl/producten/pdok-locatieserver
and 
https://github.com/PDOK/locatieserver/wiki/API-Locatieserver

Would be cool to have the plugin updated to use the new api?
PdokServicesPlugin is already updated, and could maybe be of some help.

**Comment by Lytrix** [comment]
@rduivenvoorde thanks for the information! I've added a hotfix to make the plugin working again with the new api for use in QGIS 2.*
Could you update the package on the QGIS plugin page for me? Please let me know if anything is missing.

**Comment by Lytrix** [comment]
@rduivenvoorde ready to be reviewed:
https://plugins.qgis.org/plugins/pdokbaggeocoder/version/0.6.0/

**Comment by rduivenvoorde** [comment]
@Lytrix works for me \o/

Some thoughts:

- would be better to remove the *.pyc files from the zip

- the dialog is fixed sizes, which look (to me) rather ugly in Linux, I loaded it in QtDesigner and did some shuffling, but there are som 'hidden' widgets of which I do not know what to do with it: notfoundfilename and browse_notfound. Parked them below in the orange spot:
![pdok bag geocoder_391](https://user-images.githubusercontent.com/731673/38970028-8c9c4378-4393-11e8-95e1-d80a28a21add.png)
are these needed? If not I can remove them and send you the new ui file

- in near future: would be cool to do the geocoding in a separate thread, then QGIS does not 'freeze'

**Comment by Lytrix** [comment]
@rduivenvoorde These hidden widgets are not needed. If you can send a new ui file that would be great! 
The freezing is a good point to fix soon, I will have a look at that after this hot fix.

**Comment by rduivenvoorde** [comment]
See https://github.com/Lytrix/pdokbaggeocoder/pull/5

I hide the notfoundfilename-widget because it is used to store the notfound filename?

**Comment by Lytrix** [comment]
Yes I saw this also, I will store it directly here, so it does not need to store it through the dialog: https://github.com/Lytrix/pdokbaggeocoder/blob/master/pdokbaggeocoder_dialogs.py#L234

**Comment by Lytrix** [comment]
Ok, I uploaded the new version with your PR:
http://plugins.qgis.org/plugins/pdokbaggeocoder/version/0.6.1/
